README.md

```diff
@@ -1,6 +1,7 @@
 # lab3_group5
 Lab3 repository
 
+Project Team Members:
```

Comment:
Hey, make sure that you compare to main! there have been a lot of changes!

Reply:
looks nice